client/src/utils/helpers.js

```diff
@@ -17,12 +17,12 @@ var _ = {
   eachAsync: function(obj, fn, cb){
     var c = obj.length;
     
-    var next = function(){
+    var next = function(err){
```

Comment:
in this case, the variable names are not helping very much w/ readability aren't they?